boards/README.md

```diff
@@ -12,25 +12,33 @@ issue is closed, the corresponding work item will also be moved to the
 action "Create Azure Boards Work Item" {
   uses = "azure/github-actions/boards@master"
   env = {
-		AZURE_BOARDS_ORGANIZATION = "<Azure Boards Organization Name>"
-		AZURE_BOARDS_PROJECT = "<Azure Boards Project Name>"
+		AZURE_DEVOPS_URL = "<Azure DevOps URL>"
+		AZURE_DEVOPS_ORGANIZATION = "<Azure DevOps Organization Name>"
+		AZURE_DEVOPS_PROJECT = "<Azure DevOps Project Name>"
 		AZURE_BOARDS_TYPE= "<Azure Boards Work Item Type>"
 		AZURE_BOARDS_CLOSED_STATE= "<Azure Boards Work Item State>"
 		AZURE_BOARDS_REOPENED_STATE= "<Azure Boards Work Item State>"
 	}
-  secrets = ["AZURE_BOARDS_TOKEN"]
+  secrets = ["AZURE_DEVOPS_TOKEN"]
 }
 ```
 
 ### Secrets
 
-- `AZURE_BOARDS_TOKEN` – **Mandatory**; an access token to be used when creating/updating work items.  See [Authenticate access with personal access tokens](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops) for details. 
+- `AZURE_DEVOPS_TOKEN` – **Mandatory**; an access token to be used when creating/updating work items.  See [Authenticate access with personal access tokens](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops) for details. 
 
 
 ### Environment variables
 
-- `AZURE_BOARDS_ORGANIZATION` – **Mandatory**
-- `AZURE_BOARDS_PROJECT` – **Mandatory** 
+One of `AZURE_DEVOPS_URL` or `AZURE_DEVOPS_ORGANIZATION` is mandatory.
+In case both are defined, `AZURE_DEVOPS_URL` gets preference.
+
+- `AZURE_DEVOPS_URL` – **Optional**; the fully-qualified URL to the Azure DevOps organization (eg, `https://dev.azure.com/organization` or `https://server.example.com:8080/tfs/DefaultCollection`)
```

Comment:
This would be Mandatory now

Reply:
@ethomson The URL would still be mandatory though

Reply:
fixed.